src/main/java/org/terasology/kcomputers/components/KallistiDisplayCandidateComponent.java

```diff
@@ -75,4 +62,13 @@ public void update(InputStream stream) throws IOException {
     public void setDisplay(KallistiDisplayComponent displayComponent) {
         this.display = displayComponent;
     }
+
+    @Override
+    public void copy(KallistiDisplayCandidateComponent other) {
+        this.borderThickness = other.borderThickness;
+        this.maxWidth = other.maxWidth;
+        this.maxHeight = other.maxHeight;
+        this.multiBlock = other.multiBlock;
+        this.display = other.display;
```

Comment:
This is another component, so I guess we should use at least it's `.copy()` method for a deep copy.